service/api_patch.go

```diff
@@ -288,6 +289,26 @@ func (as *APIServer) submitPatch(w http.ResponseWriter, r *http.Request) {
 	as.WriteJSON(w, http.StatusCreated, PatchAPIResponse{Patch: patchDoc})
 }
 
+func extractDisplayTasks(pairs []model.TVPair, tasks []string, variants []string,
```

Comment:
should this return model.TaskVariantPairs

Reply:
done